src/routes/user/userLogout.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  method: 'GET',
+  path: '/user/logout',
+  config: { auth: false },
```

Comment:
Should the auth be false here? 
The user will still be logged in to be able to access this route right?

Reply:
currently it is false , for testing , when we merge it with front end it will be changed

Reply:
Ok